site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
-### Review violation troubleshooting updates
-
-#### Published 2021.10.27
+### Chrome 96: Dynamic content scripts {: #m96-dynamic-content-scripts }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting/) API now supports
+[registering](/docs/extensions/reference/scripting/#method-registerContentScripts),
+[updating](/docs/extensions/reference/scripting/#method-updateContentScripts),
+[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
+a list of content scripts](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts)
```

Comment:
```suggestion
[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
a list of](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts) content scripts
```

Reply:
Ack.

Reply:
Please keep the link on 'content scripts'. It's more descriptive than 'list'. Alternatively, you could link the whole phrase.

Reply:
I'm not sure if this is what @rdcronin had in mind, but as I think about these links a bit more I think I see a clear rationale for keeping "content scripts" outside of the link. This sentence provides a list of operations that are all related to content scripts, and each one links to a relevant method. By scoping the links to the operation and having "content scripts" outside of the link text, it's more obvious to the reader that each of the items in the list operates on the term that follows the list.

At the moment I have a slight to moderate preference for this structure:

> The [`chrome.scripting`] API now supports [registering], [updating], [unregistering], and [getting
a list of] content scripts at runtime.

Reply:
> This sentence provides a list of operations that are all related to content scripts, and each one links to a relevant method. By scoping the links to the operation and having "content scripts" outside of the link text, it's more obvious to the reader that each of the items in the list operates on the term that follows the list.

Yep, exactly this : )